check_imagestreams.py

```diff
@@ -47,6 +48,9 @@ def check_imagestreams(self):
             print(f"No json files present in {IMAGESTREAMS_DIR}.")
             return 0
         for f in json_files:
+            if os.environ.get("TARGET") in ("rhel7", "centos7") and str(f).endswith("aarch64.json"):
```

Comment:
Would check for substring `aarch64` instead, in case we end up renaming the imagestream files yet again.

Reply:
It was my first proposal, but for renaming case it make sense.